ct.yaml

```diff
@@ -1,3 +0,0 @@
-remote: origin
-chart-dirs:
-    - charts
```

Comment:
Just curious, why was this removed?

Reply:
I think the ct util does it automatically; should we put it back?